packages/core/core-flows/src/common/steps/dismiss-remote-links.ts

```diff
@@ -13,18 +13,27 @@ export const dismissRemoteLinkStep = createStep(
     const link = container.resolve<RemoteLink>(
       ContainerRegistrationKeys.REMOTE_LINK
     )
+
+    // Our current revert strategy for dismissed links are to recreate it again.
```

Comment:
Should we open a ticket to support this out of the box? I think having to keep the original data in memory here is a lot more wasteful

Reply:
I think so. The only way to not do this would be to soft delete links. Would have to check if this approach was taken intentionally cc. @carlos-r-l-rodrigues

Reply:
Yeah I was thinking we do soft-deletes like everywhere else, but let's see what Carlos says

Reply:
They are soft deleted by default, even when you call the method RemoteLink.delete.
And the method create is an upsert under the hood. You can try to just call the create again with the PK and FK and it will just set the deleted_at null.
If this removes the extra fields, we'll need another method `restoreLink` or something...

Reply:
Change doesn't seem as easy, I've added a TODO on the file.